### ISSUE TITLE
Help in writing pseudocode for YOLO training

### ISSUE THREAD

**Issue by OndoyManing** [AlexeyAB/darknet#2486]
My goal is to easily explain the training process of the YOLO algorithm in layman's term. Can someone help me make the pseudocode?

**Comment by AlexeyAB** [comment]
You can read:
* https://github.com/AlexeyAB/darknet/issues/2389
* https://github.com/AlexeyAB/darknet/issues/666#issuecomment-388565246
* https://github.com/AlexeyAB/darknet/issues/796#issuecomment-388553709
* https://github.com/AlexeyAB/darknet/issues/1845#issuecomment-440283505

**Comment by OndoyManing** [comment]
@AlexeyAB Thanks for the references! I've spent almost a week in reading them, this [comment ](https://github.com/AlexeyAB/darknet/issues/796#issuecomment-388778849)seems to be the most simplified explanation of how training is done, but no matter how hard I try, I could not create an equivalent pseudocode of it. Could you help me out on this one?
